src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,58 @@
 package core.basesyntax;
 
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+
 public class WorkWithFile {
+
+    private static final Integer INDEX_OF_COUNT = 1;
+    private static final String SUPPLY = "supply";
+    private static final String BUY = "buy";
+    private static final String RESULT = "result";
+    private static final String COMA = ",";
+
     public void getStatistic(String fromFileName, String toFileName) {
+        String statistic = readFromFile(fromFileName);
+        writeToFile(toFileName, statistic);
+    }
+
+    private String readFromFile(String fileName) {
+        File file = new File(fileName);
+        List<String> text;
+        try {
+            text = Files.readAllLines(Path.of(file.toURI()));
+        } catch (IOException e) {
+            throw new RuntimeException("Can not read file", e);
+        }
+        return calculateStatistic(text);
```

Comment:
```suggestion
        return text;
```
![image](https://github.com/mate-academy/jv-work-with-file/assets/20326858/f801b896-ff6c-40d0-8375-0f335133b984)